fe/fe-core/src/main/java/com/starrocks/qe/scheduler/slot/SlotRequestQueue.java

```diff
@@ -142,8 +142,8 @@ private boolean isGroupSlotAvailable(ResourceGroup group, int numAllocatedSlotsO
             return true;
         }
 
-        int numTotalSlots = group.getConcurrencyLimit();
-        return numTotalSlots <= 0 || numAllocatedSlotsOfGroup < numTotalSlots;
+        Integer numTotalSlots = group.getConcurrencyLimit();
```

Comment:
I think the default value for concurrencyLimit should be 0 or -1, means no limit. shouldn't return null for getConcurrencyLimit.

Reply:
Yes, I agree with you. Use `Integer` could expose more bug ricks.
However, it is the logic long long ago. Maybe there is compatibility problem if I change it from `Integer` to `int`.